wbsv/ParseArgs.py

```diff
@@ -50,7 +50,11 @@ def parse_args():
                         type=natural_num, metavar='lv',
                         help='Set maximum recursion depth.')
     parser.add_argument('-d', '--dry_run', action='store_true', default=False,
-                        help='Running without saving (dry-run mode)')
+                        help='Run without saving (dry-run mode)')
+    parser.add_argument('-f', '--fake_fetch', action='store_true', default=False,
+                        help='Run without page fetching.')
+    parser.add_argument('-n', '--no_wait', action='store_true', default=False,
+                        help='Run without waiting (-d, -f only).')
```

Comment:
This newly-added part means:
- for testing, make program search within dummy sites
- when retrying and finding links, run without wait
It is true?

Reply:
And please get rid of `default=False,` .
`action='store_true'` sets false to flag by default.